tileset.go

```diff
@@ -38,6 +41,32 @@ type Tileset struct {
 	parentMap *Map
 }
 
+func readTileset(r io.Reader) (*Tileset, error) {
+	log.Debug("readTileset: reading from io.Reader")
+
+	d := xml.NewDecoder(r)
+
+	t := new(Tileset)
```

Comment:
I'd prefer to avoid `new` unless absolutely necessary, to remain consistent with the codebase.  Other suggested changes provided below.
```suggestion
	var t Tileset
```

Reply:
Your suggested change is not equivalent.
In my change, `t` would be a `*Tileset` initialized to a new one, and not `nil`.
With your suggested change, it would be a `Tileset` (not a pointer).

The alternative to `new` would be `t := &Tileset{}`.
I would suggest keeping `new`, as it is in line with the style from `tilepix.go` where it is used with `new(Map)` in `Read()`, which is where I modeled my changes from.